.github/ISSUE_TEMPLATE/work-item-template.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Work item template
+about: This issue describes a work-item ....
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+- describe purpose of work-items
+- label issue with affected components
```

Comment:
```suggestion
**List affected components**
Give an overview what components are affected by this work-item. For instance, this work-item around attestation may touch ERCC and ECC components.
**Link to feature or bug**
Please link this work-item to a feature or bug (if applicable).
```

Reply:
hmm, if we have component labels according labels (#251) do we really have to list them? Seems a bit duplicate and might lead folks _not_ to label (yet label is kind of nice as you can identify set of issues touching component which you cannot do from label).

Reply:
you are right - what I suggested here is redundant with github tags.

So you want here a grouping label to ... group a set of work-items? maybe a feature or a bug can be this kind of grouping element?

Reply:
Look at the revised template: i added a TODO header which is supposed to tell the issue creator what to do.   as you can add as many labels as you want you do not really need a grouping element here?  But having labels in a way that same-type-labels-look-similar-and-distinct-from-others would seem useful, hence my comment in #251 to either group by color or by a common prefix?